brion/blueConfig.h

```diff
@@ -74,6 +74,12 @@ class BlueConfig : public boost::noncopyable
                                       const std::string& sectionName,
                                       const std::string& key ) const;
 
+    /**
+     * @return the user and circuit file target parsers.
+     * @version 1.7
+     */
+    BRION_API Targets getTargetParsers() const;
```

Comment:
That was my first option. But getTargets is rather ambiguous, even misleading, on its own. A user may think that Targets is the list of all GIDSets or similar. Since brion::Target has a parse method I decided to include Parser in the method name, to make it clearer. I'll remove it is somebody else agrees with you.